tutorials/llama_index_knowledge_base_tutorial.ipynb

```diff
@@ -0,0 +1,402 @@
+{
```

Comment:
<p>I think this is missing a big step - how would the user even generate the granular subject of the queries? If large granular subjects are missing from the knowledge base, you could just use traditional tabular drift as well</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/734/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
<p>In this case, the subjects are the titles of each Wikipedia article in the index. Would it help to change to "title"?</p>